test/Microsoft.Azure.Devices.Edge.Test/SetupFixture.cs

```diff
@@ -142,4 +146,49 @@ public async Task BeforeAllAsync()
                 Log.CloseAndFlush();
             });
     }
+
+    // Generates a test CA cert, test CA key, and trust bundle.
+    // TODO: Remove this once iotedge init is finished?
+    public class TestCertificates
+    {
+        private string deviceId;
+        private CaCertificates certs;
+
+        TestCertificates(string deviceId, CaCertificates certs)
+        {
+            this.deviceId = deviceId;
+            this.certs = certs;
+        }
+
+        public static async Task<TestCertificates> GenerateCertsAsync(string deviceId, CancellationToken token)
+        {
+            string scriptPath = Context.Current.CaCertScriptPath.Expect(
+                () => new System.InvalidOperationException("Missing CA cert script path"));
+            (string certPath, string keyPath, string password) = Context.Current.RootCaKeys.Expect(
+                () => new System.InvalidOperationException("Missing root CA"));
+
+            await OsPlatform.Current.InstallRootCertificateAsync(certPath, keyPath, password, scriptPath, token);
```

Comment:
The CertificateAuthority constructor appears to do a similar root CA install. How is this different?

Reply:
This is using the lower-level OsPlatform instead of CertificateAuthority. Fixed to used CertificateAuthority